src/components/combo_box/combo_box.tsx

```diff
@@ -142,6 +142,16 @@ export interface _EuiComboBoxProps<T>
    * Specifies that the input should have focus when the component loads
    */
   autoFocus?: boolean;
+  /**
+   * Creates an `aria-label` attribute when using the combo box without a visible label from `<EuiFormRow />`
```

Comment:
```suggestion
   * Required when rendering without a visible label from [EuiFormRow](/#/forms/form-layouts).
```

We don't need to duplicate the name of the prop as it's applied, but just say when it is necessary